### ISSUE TITLE
[Feature]:  Support for Obsidian Publish

### ISSUE THREAD

**Issue by mascaron** [mgmeyers/obsidian-kanban#890]
### Goal or desired outcome of this feature

The kanban board is viewable when uploaded to Obsidian Publish instead of being uploaded as the raw markdown.

### Describe the feature

Ideally the kanban board would be visible as a static board.  Similar to how you can publish a kanban board in Notion as a static page.

### Can you think of any alternatives or work-arounds?

Even it exporting as a table when uploaded to Publish would be valuable.

**Comment by eholtrop** [comment]
Agreed! I want to have a kanban that I can edit normally but then show roadmaps to my users for what is coming next! (And in theory allow them to vote for what I work on next.... but thats different)
